src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -339,6 +340,13 @@ public AgentSettings LoadSettings()
                 var serviceControlManager = HostContext.GetService<IWindowsServiceControlManager>();
                 serviceControlManager.ConfigureService(agentSettings, command);
             }
+            //This will be enabled with AutoLogon code changes are tested
```

Comment:
yes the full config/unconfig exp is going in but since upgrade path is not yet done, I am not associating it with the main flow.

---
In reply to: [117651381](https://github.com/Microsoft/vsts-agent/pull/978#discussion_r117651381) [](ancestors = 117651381)